### DISCUSSION TITLE
Can you put a GraphQL Schema in Swagger?

### DISCUSSION THREAD

**Discussion by ms185730** [graphql-java/graphql-java#2406] [General]
I have a GraphQL endpoint and a couple of regular REST endpoints with springboot in the same application. The REST endpoints are documented in Swagger and I was wondering what the best way to expose the GraphQL schema to the client would be. I'm hoping to also put it in swagger with the other endpoints, is that possible?

**Comment by bbakerman** [comment]
Its possible in a silly way - in the that `/graphql` end point takes args like query and operation name and these could be Swagger described

But I dont think its possible to describe the graphql type system (the schema ) in Swagger because while elements of it look a bit like JSON schema - its not that.

graphql != REST by a large margin and Swagger is a REST documentation mechanism.
